src/mocking/server/index.ts

```diff
@@ -66,6 +66,11 @@ createServer({
     this.passthrough((request) => {
       return request.url.includes('hotjar')
     })
+
+    // allow Sentry error reporting
```

Comment:
We could probably merge that with the hotjar one, just include additional condition there